Modules/main.c

```diff
@@ -35,9 +35,11 @@
 extern "C" {
 #endif
 
+extern void _PyMem_GetDefaultRawAllocator(PyMemAllocatorEx *alloc);
```

Comment:
When I used this technique for the locale coercion APIs, I ended up causing a build failure on Cygwin: https://bugs.python.org/issue31877

Assuming that's a general concern, `PyAPI_FUNC` will be needed here as well.

Reply:
Fixed